<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="add_node_Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Node</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="enter_First_nodeName_edge_TextField">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="enter_Second_nodeName_edge_TextField">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="make_edge_Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Edge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Node 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Node 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dictionary_viewer_TextEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>350</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="visualize_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bfs_radioButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BFS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dijkstra_radioButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dijkstra's</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ballman_ford_radioButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Bellman-Ford</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dfs_radioButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DFS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allow_weight_Button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow 
Weights</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edge_weight_TextField">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dictionary</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="algorithms_results_viewer_TextEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>490</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithms Search Results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>660</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
